src/rules/noAnyRule.ts

```diff
@@ -33,7 +33,7 @@ export class Rule extends Lint.Rules.AbstractRule {
     };
     /* tslint:enable:object-literal-sort-keys */
 
-    public static FAILURE_STRING = "Type declaration of 'any' is forbidden";
+    public static FAILURE_STRING = "Type declaration of 'any' is forbidden (use the empty type '{}'?)";
```

Comment:
We have a lot of novice users, who are mailing out their PR for review, and their command-line (or even an email) reports that they have to stop and go back to fix something. For these users, there is plenty of space for more text, and this is the right UX to also explain to a novice user what they did wrong and how to fix it.

IMO the core problem here is we have one string field which must fit into a tooltip, in an IDE environment where there is other help, and also be used in the command-line interface where the context is missing.

Maybe on the command line we could also print the Description field? Though I know most descriptions were not written with this in mind. Maybe add a new field "LONG_FAILURE_STRING" or similar? @JKillian ?